projects/packages/forms/src/blocks/contact-form/variations.js

```diff
@@ -755,6 +757,228 @@ const variations = compact( [
+						fill="none"
+						className="icon-filler"
+					/>
+					<Path
+						d="M21.3501 12C21.3501 10.7574 20.3427 9.75 19.1001 9.75C17.8575 9.75 16.8501 10.7574 16.8501 12C16.8501 13.2426 17.8575 14.25 19.1001 14.25C20.3427 14.25 21.3501 13.2426 21.3501 12Z"
+						stroke="currentColor"
+						strokeWidth="1.5"
+						fill="none"
+						className="icon-filler"
+					/>
+					<Path
+						d="M19.1001 18.8636C19.1001 21.1364 17.4668 22.5 15.6001 22.5C13.7334 22.5 12.1001 21.5 12.1001 18.8636V16.5"
+						stroke="currentColor"
+						strokeWidth="1.5"
+						fill="none"
+						className="icon-filler"
+					/>
+					<Path d="M21.6001 19H20.1001H18.6001H16.6001L19.1001 16L21.6001 19Z" />
+					<Path
+						d="M12.1001 5.1364C12.1001 2.8636 10.4668 1.5 8.6001 1.5C6.73343 1.5 5.1001 2.5 5.1001 5.1364V7.5"
+						stroke="currentColor"
+						strokeWidth="1.5"
+						fill="none"
+						className="icon-filler"
+					/>
+					<Path d="M14.6001 5H13.1001H11.6001H9.6001L12.1001 8L14.6001 5Z" />
+				</>
+			),
+		},
+		innerBlocks: [
+			[ 'jetpack/form-progress-indicator', {} ],
+			[
+				'jetpack/step-container',
+				{},
+				[
+					[
+						'jetpack/form-step',
+						{ stepLabel: __( "Let's get acquainted", 'jetpack-forms' ) },
+						[
+							[
+								'jetpack/field-name',
+								{ placeholder: __( 'Jamie Smith', 'jetpack-forms' ) },
+								[
+									[ 'jetpack/label', { label: __( 'Name', 'jetpack-forms' ) } ],
+									[ 'jetpack/input' ],
+								],
+							],
+							[
+								'jetpack/field-email',
+								{
+									required: true,
+									placeholder: __( 'jamie.smith@example.com', 'jetpack-forms' ),
+								},
+								[
+									[ 'jetpack/label', { label: __( 'Email Address', 'jetpack-forms' ) } ],
+									[ 'jetpack/input' ],
+								],
+							],
+							[
+								'jetpack/field-select',
+								{
+									options: [
+										__( 'I need help', 'jetpack-forms' ),
+										__( "I'm interested in your services", 'jetpack-forms' ),
+										__( 'Just exploring', 'jetpack-forms' ),
+									],
+								},
+								[
+									[
+										'jetpack/label',
+										{ label: __( 'What brings you here today?', 'jetpack-forms' ) },
+									],
+									[
+										'jetpack/input',
+										{ type: 'dropdown', placeholder: __( 'Select one option', 'jetpack-forms' ) },
+									],
+								],
+							],
+						],
+					],
+					[
+						'jetpack/form-step',
+						{ stepLabel: __( 'How can we help?', 'jetpack-forms' ) },
+						[
+							[
+								'jetpack/field-textarea',
+								{},
+								[
+									[
+										'jetpack/label',
+										{ label: __( 'What do you need help with?', 'jetpack-forms' ) },
+									],
+									[ 'jetpack/input', { type: 'textarea' } ],
+								],
+							],
+							[
+								'jetpack/field-select',
+								{
+									options: [
+										__( 'ASAP', 'jetpack-forms' ),
+										__( 'This week', 'jetpack-forms' ),
+										__( 'No rush', 'jetpack-forms' ),
```

Comment:
I'm not sure how to word those options better, but "ASAP" and "No rush" look too casual for me, specially "ASAP"

Reply:
cc @ilonagl